sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -683,7 +683,7 @@ pi_result cuda_piPlatformGetInfo(pi_platform platform,
   switch (param_name) {
   case PI_PLATFORM_INFO_NAME:
     return getInfo(param_value_size, param_value, param_value_size_ret,
-                   "NVIDIA CUDA");
+                   "NVIDIA CUDA BACKEND");
```

Comment:
Is there any standard to what this name should be? How including "BACKEND" is helpful?

Reply:
This differentiates the NVIDIA CUDA OpenCL platform from the NVIDIA CUDA backend we have written. That means we can search for the `"CUDA BACKEND"` string when looking for the PI backend, and not accidentally select the OpenCL one.